File: examples/go/quran/verses/getVersesOfAChapter.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/quran"
)

// Lookup the verses (ayahs) for a chapter including
// translation, interpretation and breakdown by individual
// words.
func GetVersesOfAchapter() {
	quranService := quran.NewQuranService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := quranService.Verses(&quran.VersesRequest{
		Chapter: 1,
	})
	fmt.Println(rsp, err)
}

File: examples/js/quran/verses/getVersesOfAChapter.js
const { QuranService } = require("m3o/quran");

// Lookup the verses (ayahs) for a chapter including
// translation, interpretation and breakdown by individual
// words.
async function getVersesOfAchapter() {
  let quranService = new QuranService(process.env.MICRO_API_TOKEN);
  let rsp = await quranService.verses({
    chapter: 1,
  });
  console.log(rsp);
}

getVersesOfAchapter();
